src/gesture.js

```diff
@@ -512,6 +549,9 @@ export class Gestures {
     this.pending_[index] = 0;
     if (!this.ready_[index]) {
       this.recognizers_[index].acceptCancel();
+      if (this.recognizers_[index].getType() == 'tapzoom') {
```

Comment:
Is `tapzoom` the only two-tap gesture? If any new gestures with two sequenced taps will be added in the future, you may want to add a comment here stating that they'll need to be added to this if statement. If I understand correctly.

It would be nice to enforce that through code as well, but that effort may not be worth it

Reply:
Lost a comment here. `'tapzoom'` is the event with the longest wait time. Basically if we get to a tapzoom, the double tap should have already fired or have been canceled.